src/core/Akka.Streams/reference.conf

```diff
@@ -147,9 +147,9 @@ akka {
     }
   
     serialization-bindings {
-      "Akka.Streams.Dsl.SinkRefImpl, Akka.Streams"         = akka-stream-ref
-      "Akka.Streams.Dsl.SourceRefImpl, Akka.Streams"       = akka-stream-ref
-      "Akka.Streams.Dsl.IStreamRefsProtocol, Akka.Streams" = akka-stream-ref
+      "Akka.Streams.Implementation.StreamRef.SinkRefImpl, Akka.Streams"         = akka-stream-ref
```

Comment:
Should note that this is not a breaking change for v1.4.0-beta1 users, as the serializer manifests automatically handle this change in namespaces on either end without any breakages.